Source/WebCore/css/ComputedStyleExtractor.cpp

```diff
@@ -357,12 +357,17 @@ static RefPtr<CSSValue> valueForNinePieceImage(CSSPropertyID propertyID, const N
     return createBorderImageValue(WTFMove(imageValue), WTFMove(imageSlices), WTFMove(borderSlices), WTFMove(outset), WTFMove(repeat));
 }
 
-static Ref<CSSPrimitiveValue> fontSizeAdjustFromStyle(const RenderStyle& style)
+static Ref<CSSValue> fontSizeAdjustFromStyle(const RenderStyle& style)
```

Comment:
It looks like every codepath through this function returns a CSSPrimitiveValue, so it looks like there's no need to make the return type more generic